sunpy/time/time.py

```diff
@@ -126,6 +126,14 @@ def _astropy_time(time):
     return time if isinstance(time, astropy.time.Time) else astropy.time.Time(parse_time(time))
 
 
+def _parse_dt64(dt):
+    """
+    Parse a single numpy datetime64 object
+    """
+    # Validate (in an agnostic way) that we are getting a datetime rather than a date
+    return datetime(*(dt.astype(datetime).timetuple()[:6]))
```

Comment:
the first 6 elements of this tuple being `year, month, day, hour, min, sec`? what about sub-second numbers?

Reply:
Again, I have no idea, this is just copied from the original `datetime64` parsing code here: https://github.com/sunpy/sunpy/pull/2370/files#diff-e4d7773dc71c1ca30c1741901255ba37L170